examples/azure-linux-vm-multi-disks/main.tf

```diff
@@ -3,30 +3,26 @@
 # Environment variables can also be used https://www.terraform.io/language/values/variables#environment-variables
 
 resource "azurerm_managed_disk" "managed_disk" {
-  name                 = var.managed_disk_name
-  location             = data.azurerm_resource_group.rg.location
-  resource_group_name  = var.azurerm_resource_group_name
-  storage_account_type = var.managed_disk_storage_account_type
-  create_option        = var.managed_disk_create_option
-  disk_size_gb         = var.managed_disk_size_gb
-  image_reference_id   = var.managed_disk_image_reference_id
+  name                 = "managed_disk"
+  location             = "eastus"
+  resource_group_name  = "kinder-testing"
+  storage_account_type = "Standard_LRS"
+  create_option        = "Empty"
+  disk_size_gb         = 8
   tags                 = var.tags
```

Comment:
Add real values instead of using a variable. 
ex:
  tags = {
    "Owner"    = "user@company.com"
  }